contrib/mock_patch_checker.py

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env python3.6
+import ast
+import sys
+
+
+class MockChecker(ast.NodeVisitor):
+    def __init__(self):
+        self.errors = 0
+        self.init_module_imports()
+
+    def init_module_imports(self):
+        self.imported_patch = False
+        self.imported_mock = False
+
+    def check_files(self, files):
+        for file in files:
+            self.check_file(file)
+
+    def check_file(self, filename):
+        self.current_filename = filename
+        try:
+            with open(filename, 'r') as fd:
+                try:
+                    file_ast = ast.parse(fd.read())
+                except SyntaxError as error:
+                    print("SyntaxError on file %s:%d" % (filename, error.lineno))
+                    return
```

Comment:
Do you want to increase self.errors here so that the pre-commit hook exits with non-zero value?

Reply:
Hmm. We don't have that on paasta. I think it is "OK", other pre-commit hooks will catch this and fail way before.